src/brogue/Items.c

```diff
@@ -2354,6 +2358,30 @@ void itemDetails(char *buf, item *theItem) {
                 strcat(buf, buf2);
             }
 
+            if (theItem->lastUsed[0] > 0 && theItem->lastUsed[1] > 0 && theItem->lastUsed[2] > 0) {
+                turnsSinceLastUse[0] = rogue.playerTurnNumber - theItem->lastUsed[0];
+                turnsSinceLastUse[1] = rogue.playerTurnNumber - theItem->lastUsed[1];
+                turnsSinceLastUse[2] = rogue.playerTurnNumber - theItem->lastUsed[2];
```

Comment:
style: Worth calculate these in one place? Or dropping them entirely?

Reply:
I'll inline the local